web/src/test/java/it/pagopa/selfcare/onboarding/web/controller/InstitutionV2ControllerTest.java

```diff
@@ -54,6 +54,25 @@ void onboardingProductAsync(@Value("classpath:stubs/onboardingProductsDtoWithout
         verifyNoMoreInteractions(institutionServiceMock);
     }
 
+    @Test
+    void onboardingProductForAggregatorAsync(@Value("classpath:stubs/onboardingProductsDtoWithAggregates.json") Resource onboardingDto) throws Exception {
+        // given
+        String institutionId = "institutionId";
+        String productId = "productId";
+        // when
+        mvc.perform(MockMvcRequestBuilders
+                        .post(BASE_URL + "/onboarding", institutionId, productId)
+                        .content(onboardingDto.getInputStream().readAllBytes())
+                        .contentType(APPLICATION_JSON_VALUE)
+                        .accept(APPLICATION_JSON_VALUE))
+                .andExpect(status().isCreated())
+                .andExpect(content().string(emptyString()));
+        // then
+        verify(institutionServiceMock, times(1))
+                .onboardingProductV2(any(OnboardingData.class));
```

Comment:
you could verify if onboardinData has aggregates not empty

Reply:
I would have done the unit tests in the next implementation task, This is just a skeleton